src/server.js

```diff
@@ -102,6 +105,16 @@ function jsonExporter(req, res, next) {
         },
       );
     })
+    .then((content) => {
+      return run(
+        `(. | .preview_image_link."@id") |= sub("${config.settings.apiPath}";"")`,
```

Comment:
It is not needed to use `..` since the only preview_image_link we need for the preview image in content creator is on the top level.

`. | .preview_image_link`

With `..` it would search for any value of object key "preview_image_link" in all child objects or arrays.

This is a potential source of crashes since there are objects with the key "preview_image_link" that are structured in a different way, so it's easier to just stay on the top level.